src/pages/home/report/ReportActionsList.js

```diff
@@ -1,20 +1,22 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import {ActivityIndicator, View} from 'react-native';
+import {View, Animated} from 'react-native';
```

Comment:
I'm curious why we aren't using `react-native-reanimated` esp. since it's already installed 

>  Reanimated aims to provide ways of offloading animation and event handling logic off of the JavaScript thread and onto the UI thread
https://docs.swmansion.com/react-native-reanimated/docs/#motivation

https://github.com/Expensify/App/blob/a0c4e83085f25492fabec48fee43f6ad4f9dc52e/package.json#L99

Reply:
We can but it is a simple animation so does not matter much.

Reply:
I agree. My only reasoning is that we're using the built-in `Animated` throughout the app.

If we don't use `Reanimated` now, it's unlikely we'll start using it anytime soon 🤷

Reply:
Okay quick note, I did try `renanimated` but I it didn't work with `react-native-web`. The fade animations worked fine on mobile apps but not on web. One more thing was if I added the `Animated...entering...exiting` effect, it had issues with scroll and a noticeable lag while loading new items.

Reply:
> didn't work with react-native-web
it had issues with scroll and a noticeable lag while loading new items

Reanimated supports `react-native-web`
https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/web-support

Maybe we didn't configure it properly? 

But since you're already tried and faced lag on native, we could leave it for now.